forloop_modules/globals/local_variable_handler.py

```diff
@@ -256,11 +257,11 @@ def update_variable(self, name, value, additional_params: dict = None, project_u
                 "variable_uid": variable["uid"],
                 "name": name,
                 "value": value,
-                "is_result": variable["is_result"]  # TODO: Remove when PrototypeJobs are implemented
+                "is_result": is_result if is_result is not None else variable["is_result"]  # TODO: Remove when PrototypeJobs are implemented
```

Comment:
please define this ternary operator before the dict + assign to different variable (more readable)